fighters/robot/src/opff.rs

```diff
@@ -18,6 +18,31 @@ unsafe fn gyro_dash_cancel(boma: &mut BattleObjectModuleAccessor, status_kind: i
     }
 }
 
+unsafe fn uspecial_cancels(boma: &mut BattleObjectModuleAccessor, situation_kind: i32, frame: f32) {
+    if WorkModule::is_flag(boma, *FIGHTER_ROBOT_STATUS_BURNER_FLAG_PUSH_B_BUTTON)
+    && situation_kind == *SITUATION_KIND_AIR
+    && frame > 11.0 {
+		if boma.is_button_on(Buttons::Guard) {
+			WorkModule::unable_transition_term_group(boma, *FIGHTER_STATUS_TRANSITION_GROUP_CHK_AIR_ESCAPE);
+			ControlModule::clear_command_one(boma, *FIGHTER_PAD_COMMAND_CATEGORY1, *FIGHTER_PAD_CMD_CAT1_AIR_ESCAPE);
+			StatusModule::change_status_request_from_script(boma, *FIGHTER_STATUS_KIND_FALL, false);
+			//let vec = Vector3f{x: 0.0, y: -0.2, z: 0.0};
+			//KineticModule::add_speed(boma, &vec);
+		} else if boma.is_button_on(Buttons::Attack) {
+			StatusModule::change_status_request_from_script(boma, *FIGHTER_ROBOT_STATUS_KIND_SPECIAL_HI_ATTACK, false);
+		}
+	}	
+}
```

Comment:
this is purely styling, but there is some weird double-tab spacing here. Make sure your development environment is set up to convert tabs to spaces, because tabs are probably expanded differently on github than your environment